flexmeasures/utils/config_utils.py

```diff
@@ -72,11 +72,27 @@ def check_app_env(env: str | None):
 def read_config(app: Flask, custom_path_to_config: str | None):
     """Read configuration from various expected sources, complain if not setup correctly."""
 
-    check_app_env(app.env)
+    flexmeasures_env = "production"
```

Comment:
This default is probably correct, or at least better than no default (what was the default before this PR?) but the place to use it seems not optimal.

Maybe we could set "production" as a default in `utils.config_defaults`.

Reply:
changed the hardcoded string to a constant that lives in the config_defaults. FLASK_ENV also defaulted to production: https://flask.palletsprojects.com/en/1.1.x/config/#environment-and-debug-features